example_data/config.yaml

```diff
@@ -0,0 +1,30 @@
+input_file: "example_data/introducing-mozilla-ai-investing-in-trustworthy-ai.html"
+output_folder: "example_data/"
+text_to_text_model: "allenai/OLMoE-1B-7B-0924-Instruct-GGUF/olmoe-1b-7b-0924-instruct-q8_0.gguf"
+text_to_speech_model: "parler-tts/parler-tts-mini-v1"
+text_to_text_prompt: |
+  You are a podcast scriptwriter generating engaging and natural-sounding conversations in JSON format. 
+  The script features the following speakers:
+  {SPEAKERS}
+  Instructions:
+  - Write dynamic, easy-to-follow dialogue.
+  - Include natural interruptions and interjections.
+  - Avoid repetitive phrasing between speakers.
+  - Format output as a JSON conversation.
+  Example:
+  {
+    "Speaker 1": "Welcome to our podcast! Today, we're exploring...",
+    "Speaker 2": "Hi! I'm excited to hear about this. Can you explain...",
+    "Speaker 1": "Sure! Imagine it like this...",
+    "Speaker 2": "Oh, that's cool! But how does..."
+  }
+sampling_rate: 44100
+speakers:
+  - id: 1
+    name: Laura
+    description: The main host. She explains topics clearly using anecdotes and analogies, teaching in an engaging and captivating way.
+    tone: Laura's voice is exciting and fast in delivery with very clear audio and no background noise.
```

Comment:
Related to [this](https://github.com/mozilla-ai/document-to-podcast/pull/34/files#r1881905039), should we change the `tone` to `profile` or something that fits both the Oute and Parler case?

Reply:
Do you think `tone: female_1` looks too bad?

Reply:
hmm, yeah I feel like `tone` implies a natural description or at least a descriptive word, not an id from a predefined list. I feel like `profile` is a bit more ambiguous.

Reply:
What do you think of voice_profile? I just want something that indicates this is about the text to speech part

Reply:
voice_profile sounds good to me!